grakn-kb/src/test/java/ai/grakn/kb/internal/concept/RelationshipTest.java

```diff
@@ -311,4 +314,17 @@ public void whenAttributeLinkedToRelationshipIsInferred_EnsureItIsMarkedAsInferr
         RelationshipImpl.from(relationship).attributeInferred(attribute);
         assertTrue(relationship.relationships().findAny().get().isInferred());
     }
+
+    @Test
+    public void whenAddingRelationshipWithNoRolePlayers_Throw(){
```

Comment:
Just double checking, do we already have a test which verifies the happy path?

Reply:
Yep we do.